MAUI/Autocomplete/Selection.md

```diff
@@ -0,0 +1,222 @@
+
+## Single selection
+
+The `Autocomplete` allows user to select an item by entering the value using keyboard, then selecting from the drop-down list by clicking the item. The selected item can be retrieved from the `SelectedItem` property.
+
+{% tabs %}
+{% highlight c# %}
+
+//Model.cs
+public class SocialMedia
+{
+    public string Name { get; set; }
+    public int ID { get; set; }
+}
+
+//ViewModel.cs
+public class SocialMediaViewModel
+{
+    public ObservableCollection<SocialMedia> SocialMedias { get; set; }
+    public SocialMediaViewModel()
+    {
+        this.SocialMedias = new ObservableCollection<SocialMedia>();
+        this.SocialMedias.Add(new SocialMedia() { Name = "Facebook", ID = 0 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Google Plus", ID = 1 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Instagram", ID = 2 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "LinkedIn", ID = 3 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Skype", ID = 4 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Telegram", ID = 5 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Televzr", ID = 6 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Tik Tok", ID = 7 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Tout", ID = 8 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Tumblr", ID = 9 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Twitter", ID = 10 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Vimeo", ID = 11 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "WhatsApp", ID = 12 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "YouTube", ID = 13 });
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfAutoComplete
+    x:Name="autocomplete"
+    WidthRequest="250"
+    ItemsSource="{Binding SocialMedias}"
+    DisplayMemberPath="Name"
+    TextMemberPath="Name" />
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAutocomplete autocomplete = new SfAutoComplete();
+SocialMediaViewModel socialMediaViewModel = new SocialMediaViewModel();
+autocomplete.BindingContext = socialMediaViewModel;
+autocomplete.ItemsSource = SocialMediaViewModel.SocialMedias;
+autocomplete.DisplayMemberPath = "Name";
+autocomplete.TextMemberPath = "Name";
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete with single selection mode](Images/Selection/SingleSelection.png)
+
+## Selection changed notification
+
+When an item is selected from the drop-down list, the `SelectionChanged` event is triggered. The `SelectionChanged` event contains the newly selected and previously selected item in the `CurrentSelection` and `PreviousSelection` properties. The `SelectionChanged` contains the following properties:
```

Comment:
When an item is selected from the drop-down list, the `SelectionChanged` event is triggered. The `SelectionChanged` event contains the newly selected and previously selected items in the `CurrentSelection` and `PreviousSelection` properties. The `SelectionChanged` contains the following properties: